cmd/trigger_ui/templates/upload-master.gohtml

```diff
@@ -0,0 +1,208 @@
+                <textarea class="w-full border" name="episode_description"></textarea>
+            </div>
+
+            <div class="flex">
+                <label class="inline-block w-20 flex-none">Tags</label>
+                <div class="multiselect flex flex-row-reverse w-full border border-[#6b7280]">
+                    <input list="TagsDatalist" name="tags" class="w-full border border-none outline-none border-box"
+                        type="text">
+                    <datalist id="TagsDatalist">
+                        {{range $element := .TagsDatalist}}
+                        <option value="{{$element}}"></option>
+                        {{end}}
+                    </datalist>
+                </div>
+            </div>
+
+            <div class="flex">
+                <label class="inline-block w-20 flex-none" for="persons_appearing">Persons appearing</label>
+                <div class="multiselect flex flex-row-reverse w-full border border-[#6b7280]">
+                    <input list="PersonsDatalist" name="persons"
+                        class="w-full border border-none outline-none border-box" type="text">
+                    <datalist id="PersonsDatalist">
+                        {{range $element := .PersonsDatalist}}
+                        <option value="{{$element}}"></option>
+                        {{end}}
+                    </datalist>
+                </div>
+            </div>
+
+
+
+            <div class="flex">
+                <label class="inline-block w-20 flex-none" for="sender_email">Email</label>
+                <textarea class="w-full border" name="sender_email" required></textarea>
+            </div>
+
+            <div class="flex">
+                <label class="inline-block w-20 flex-none" for="Direct to Playback">Direct to Playback</label>
+                <select class="w-full border" name="Directto Playback">
+                    <option value="Yes">Yes</option>
+                    <option value="No" selected="selected">No</option>
+                </select>
+            </div>
+
+            <div class="flex">
+                <label class="inline-block w-20 flex-none" for="path">
+                    path for file
+                </label>
+                <select class="w-full border" name="path" required>
+                    {{range $element := .fileDirectory}}
+                    <option value="{{$element}}">{{$element}}</option>
+                    {{end}}
+                </select>
+            </div>
+
+            <input id="submit"
+                class="cursor-pointer rounded-md bg-[#6A64F1] py-3 px-8 text-center text-base font-semibold text-white outline-none"
+                type="submit" value="Start export">
+        </div>
+
+        <span type="hidden" id="arrayData"></span>
+
+    </form>
+
+    <script>
+
+        for (element of document.querySelectorAll(".multiselect")) {
```

Comment:
Does this even work? Element is undefined. use const element of document.querySelector...